docs/workflows/transitions/preprocess/preprocess/make_predictions.py

```diff
@@ -58,17 +97,27 @@ def above_threshold_or_optional(prediction):
                 is_optional = not field_config[label].get('required', True)
 
                 return (threshold['automated'] <= confidence) or (is_optional and confidence < threshold['low'])
-            
-            all_above_or_optional = all(map(above_threshold_or_optional, filter_by_top1(predictions)))
+
+            lines = any([is_line(field_config, p) for p in predictions])
+            all_above_or_optional = not lines and all(map(above_threshold_or_optional, filter_by_top1(predictions)))
             has_all_required_labels = required_labels(field_config) <= set(map(lambda p: p['label'], predictions))
             skip_validation = has_all_required_labels and all_above_or_optional
-            
+
             logging.info(f'All predictions above threshold (or optional): {all_above_or_optional}')
             logging.info(f'All required labels exist: {has_all_required_labels}')
             
             # Filter out optional fields where confidence < low
             predictions = filter_optional_fields(predictions, field_config)
+
+            if old_ground_truth:
+                predictions = merge_predictions_and_gt(predictions, old_ground_truth, field_config)
+                logging.info(f'updated predictions: {predictions}')
+
             output = {'predictions': predictions}
+        elif old_ground_truth:
+            # These will not have any confidence levels
```

Comment:
Are we sure that the validation UI and post-process image handles predictions without confidences?